public/css/index.css

```
@@ -19,6 +19,6 @@
 
 @layer components, legacy, paper;
 
-@import url('../styles.css?v=87')      layer(components);
+@import url('../styles.css?v=93')      layer(components);
```

Stylelint (17.10.0)

[error] Expected "url('../styles.css?v=93')" to be "'../styles.css?v=93'" (import-notation)

(import-notation)